tinker/templates/e-announcements/form.html

```diff
@@ -32,8 +32,9 @@
 {% block main_content %}
     <div class="content">
         <div class="container-fluid row">
+            {# The onsubmit code is to prevent the form submitting multiple times. #}
             <form id="eannouncementform" action="{{ url_for('EAnnouncementsView:submit') }}"
-                      method="post" enctype="multipart/form-data">
+                      method="post" enctype="multipart/form-data" onsubmit="return (typeof submitted == 'undefined') ? (submitted = true) : !submitted">
```

Comment:
so this would return `!submitted` if `!(typeof submitted == 'undefined)` ? Isn't this backwards?

Reply:
I believe it equates to this:

if (typeof submitted == 'undefined'):
     return submitted = true
else:
     return !submitted

Reply:
Ah I see now— I didn't read the description enough, so I thought it seemed backwards